test/regress/regress0/error.cvc

```diff
@@ -1,8 +1,7 @@
+% ERROR-SCRUBBER: sed -e '/^[[:space:]]*$/d'
```

Comment:
I admit this one is a hack. :)

These two tests fail on Windows, but not Unix, and my *guess* is that the problem is due to line ending differences (i.e. on Unix we strip/ignore the empty lines correctly, but we don't on Windows, or something like that). It was taking more time to figure this one out than all the other issues combined, so I opted for the obvious (but less than ideal) fix for the sake of getting the tests to pass 100% on Windows.

I don't think we should unconditionally strip empty lines since we'd have to "fix" (i.e. mangle) all the existing tests even though they aren't causing problems. Worse, it would make creation of new tests confusing: notice that in this little hack, we have to mangle the line number as well to correspond to the fact that we've removed the empty line...